base_station_nodes: [WayPoint0]
local_storage_node: WayPoint1
charging_station_node: WayPoint2
robot_ids: [robot00]
